### ISSUE TITLE
Enhance Image Handling for Library Agnosticism

### ISSUE THREAD

**Issue by sabir-aspose** [fileformat-words/FileFormat.Words-for-.NET#5]
OpenXML SDK stores images in drawing objects. We propose to make the custom Image object within FileFormat.Words.IElements library agnostic, enabling the storage of images as byte[] data, along with pixel dimensions using the default 96 DPI. This change empowers API users to utilize any image library of their preference for tailored image processing.

**Comment by sabir-aspose** [comment]
Resolved in 23.10.0. Now, API stores Images as byte[] and user can use an image library to further work on it. Below is the example to parse all images from a Word Document using FileFormat.Words, decode those images using SkiSharp and then save to disk as image1.png, image2.png and so on:

```csharp
using System;
using System.Threading.Tasks;
using FileFormat.Words;
using FileFormat.Words.IElements;
using System.Collections.Generic;
using SkiaSharp;
using System.IO;

namespace FileFormat.Words.Examples
{
    class Program
    {
        static void Main(string[] args)
        {
            Document doc = new Document("WordDocumentWithImages.docx");
            Body body = new Body(doc);
            int counter = 1;
            
            foreach (Image image in body.Images)
            {
                byte[] imageBytes = image.ImageData;

                using (SKBitmap skBitmap = SKBitmap.Decode(imageBytes))
                using (SKImage skImage = SKImage.FromBitmap(skBitmap))
                {
                    SKData encoded = skImage.Encode(SKEncodedImageFormat.Png, 100);
                    using (Stream stream = encoded.AsStream())
                    {
                        string imageName = $"Image{counter}.png";
                        using (FileStream fileStream = new FileStream(imageName, FileMode.Create))
                        {
                            stream.CopyTo(fileStream);
                            counter++;
                            Console.WriteLine("Image Copied");
                        }
                    }
                }
            }
        }
    }
}
```

The issue will be closed after adding related examples and articles.

**Comment by openize-words** [comment]
already resolved.
